MagazineLayout/Public/MagazineLayout.swift

```diff
@@ -602,6 +629,12 @@ public final class MagazineLayout: UICollectionViewLayout {
   private let modelState = ModelState()
   private var cachedCollectionViewWidth: CGFloat?
 
+  // These properties are used to prevent scroll jumpiness due to self-sizing after rotation; see
+  // comment in `invalidationContext(forPreferredLayoutAttributes:withOriginalAttributes:)` for more
+  // details.
+  private var lastSizedElementMinY: CGFloat?
+  private var lastSizedElementPreferredHeight: CGFloat?
```

Comment:
Should we nil out these two values in `prepare` when we have a `rebuildSectionModels` prepare action?

Reply:
That seems like a good idea. I think we should also clear it if we have an `.updateLayoutMetrics` prepare action, since that indicates new sizing modes potentially. I've pushed a commit to implement this, thanks @brynbodayle!